.travis.yml

```diff
@@ -45,7 +45,11 @@ install:
 
 script:
   - yarn lint && yarn validate-changelog && yarn check-modified && yarn
-    build:prod && yarn test:setup && yarn test && yarn test:report
+    build:prod && yarn test:setup && yarn test
```

Comment:
The problem here is that we're using `&&` here to only continue when the previous command exited cleanly. This is great to ensure we fail fast, but we should run `test:report` whether or not the tests passed.